units/sys-fs-fuse-connections.mount

```diff
@@ -11,6 +11,7 @@ Documentation=https://www.kernel.org/doc/Documentation/filesystems/fuse.txt
 Documentation=http://www.freedesktop.org/wiki/Software/systemd/APIFileSystems
 DefaultDependencies=no
 ConditionPathExists=/sys/fs/fuse/connections
+ConditionCapability=CAP_SYS_ADMIN
```

Comment:
Shouldn't we add `ConditionCapability=CAP_SYS_ADMIN` to [dev-mqueue.mount](https://github.com/systemd/systemd/blob/master/units/dev-mqueue.mount) too?
```
# systemctl status dev-mqueue.mount
WARNING: terminal is not fully functional
● dev-mqueue.mount - POSIX Message Queue File System
   Loaded: loaded (/usr/lib/systemd/system/dev-mqueue.mount; static; vendor preset: enabled)
   Active: failed (Result: exit-code) since Thu 2016-02-11 02:23:19 UTC; 3min 45s ago
    Where: /dev/mqueue
     What: mqueue
     Docs: man:mq_overview(7)
           http://www.freedesktop.org/wiki/Software/systemd/APIFileSystems
  Process: 15 ExecMount=/usr/bin/mount mqueue /dev/mqueue -t mqueue (code=exited, status=32)

Feb 11 02:23:19 nspawn-root mount[15]: mount: permission denied
Feb 11 02:23:19 nspawn-root systemd[1]: dev-mqueue.mount: Child 15 belongs to dev-mqueue.mount
Feb 11 02:23:19 nspawn-root systemd[1]: dev-mqueue.mount: Mount process exited, code=exited status=32
Feb 11 02:23:19 nspawn-root systemd[1]: dev-mqueue.mount: Changed mounting -> failed
Feb 11 02:23:19 nspawn-root systemd[1]: dev-mqueue.mount: Job dev-mqueue.mount/start finished, result=failed
Feb 11 02:23:19 nspawn-root systemd[1]: Failed to mount POSIX Message Queue File System.
Feb 11 02:23:19 nspawn-root systemd[1]: dev-mqueue.mount: Unit entered failed state.
```